src/client/conn.rs

```diff
@@ -704,6 +707,20 @@ impl Builder {
         self
     }
 
+    /// Set whether to support preserving original header order.
+    ///
+    /// Currently, this will record the order in which headers are received, and store this
+    /// ordering in a private extension on the `Response`. It will also look for and use
+    /// such an extension in any provided `Request`.
+    ///
+    /// Note that this setting does not affect HTTP/2.
+    ///
+    /// Default is false.
+    pub fn http1_preserve_header_order(&mut self, enabled: bool) -> &mut Builder {
```

Comment:
Yea, nice, exactly what I meant, thanks! (We might find that `cfg(feature = "ffi")` needs to be sprinkled around, sorry about that pain.)

Reply:
Done